### PR TITLE
Replace model benchmarking to tests

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Replace models benchmarking from `examples/experimental/onnx` to `tests/onnx/benchmarking`

### Reason for changes

This division is necessary to not confuse the users. After merging this PR NNCF will have ONNX PTQ examples for external users in  correspond directory - `examples/experimental/onnx`, while the scripts for running post-training quantization becnhmarking for many models supported with AC config in `tests/onnx/benchmarking`.

### Related tickets

89910

### Tests

No tests are needed

### CODE REVIEW

**Comment by kshpv** [comment]
@vshampor @AlexKoff88 please take a look

**Comment by kshpv** [comment]
@MaximProshin please take a look

File: examples/experimental/onnx/yolov5/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+$ git init
+$ git remote add origin https://github.com/ultralytics/yolov5.git
+$ git fetch origin
+$ git checkout 34df5032a7d2e83fe3d16770a03bd129b115d184
+$ git apply 0001-Add-NNCF-ONNX-PTQ-example-notebook.patch
+```
+
+## Run NNCF ONNXRuntime PTQ
+
+After [prerequisite](#prerequisite) is done, you can find `run_notebook.ipynb` notebook file in `examples/experimental/onnx/yolov5`. If you finish running all notebook cells, you will obtain the following PTQ benchmark results.
```
Comment (AlexKoff88) [inline examples/experimental/onnx/yolov5/README.md:19]: Why do we need to keep Jupyter notebook in the patch? 
The preferable approach for me would be to have a notebook here that does everything. Is it possible?

Comment (kshpv) [reply examples/experimental/onnx/yolov5/README.md:19]: I assume to work on examples later, now would like to concentrate on the models coverage extension.

**Review by AlexKoff88 — APPROVED** [review]

File: examples/experimental/onnx/yolov5/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Prerequisite
+
+1. Follow the [installation step](#installation)
+2. Clone [Yolov5 repository](https://github.com/ultralytics/yolov5.git) and patch it.
+
+```bash
+$ cd examples/experimental/onnx/yolov5
+$ git init
+$ git remote add origin https://github.com/ultralytics/yolov5.git
```
[the remark this answers is not in the document]
Comment (kshpv) [reply examples/experimental/onnx/yolov5/README.md:11]: there is already specified commit
`$ git checkout 34df5032a7d2e83fe3d16770a03bd129b115d184`

**Pull request merged**